contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -3562,8 +3563,9 @@ rename_procfunc_update_bbf_catalog(RenameStmt *stmt)
 	initStringInfo(&new_funcsign);
 	appendStringInfoString(&new_funcsign, stmt->newname);
 	appendStringInfoString(&new_funcsign, strrchr(funcsign, '('));
+	namestrcpy(&newname_data, stmt->newname);
```

Comment:
Where is this memory being freed?

Reply:
When `NameData newname_data` goes out of scope. We are copying the string into the static array in the NameData struct.

Reply:
I missed this I thought namestrcpy would do a `palloc()`. This looks good to me